README.md

```diff
@@ -346,8 +346,36 @@ lookup (for physical measurements and biospecimen donation) as well as a
 Participant Work Queue. Any of the above parameters can be provided as a query parameter to do
 an exact match. Examples:
 
-    GET /ParticipantSummary?hpoId=PITT
-    GET /ParticipantSummary?hpoId=PITT&state=PIIState_MA
+    GET /ParticipantSummary?awardee=PITT
+    GET /ParticipantSummary?awardee=PITT&state=PIIState_MA
+
+    GET /ParticipantSummary?organization=PITT_UPMC 
+    GET /ParticipantSummary?site=hpo-site-UPMC
+
+The participant summary API supports filtering and sorting on last modified time.
+The order results are returned is...
+* last modified time
+* participant ID (ascending)
+
+Example:
+
+    GET /ParticipantSummary?awardee=PITT&last_modified=ge2018-02-29
+See FHIR search prefixes below
+
+Synchronize Participant Summary last modified link.
+This allows Awardees to stay up-to-date
+with newly-arrived summaries. The return value is a FHIR History [Bundle](http://hl7.org/fhir/bundle.html)
+where each entry is a `ParticipantSummary` document.
+
+The Bundle's `link` array will include a link with relation=`next` if more results are available immediately.
```

Comment:
You could show an example of what this looks like here.

Reply:
Ping.

Reply:
done